documents/gettingStarted/networking/swarm.html

```diff
@@ -303,6 +303,12 @@
 $ cd netplugin; make demo
 $ vagrant ssh netplugin-node1
 </code></pre>
+<p>You need to set this exact DOCKER_HOST variable on all 3 nodes before you proceed to next steps.</p>
+<pre class="highlight plaintext"><code>export DOCKER_HOST=192.168.2.10:2375
+</code></pre>
+<p>This will make sure that everytime when someone is executing docker / netctl commands,
+It will contact docker demon running on 192.168.2.10 node and will not contact local
+docker deamon.</p>
```

Comment:
We want user to set DOCKER_HOST before he / she tries to create network and containers. So in that flow I thought of adding comment there.